### DISCUSSION TITLE
Application for GitHub Copilot Rejected Despite Valid School Certificate

### DISCUSSION THREAD

**Discussion by OriyaU** [community/community#128055] [GitHub Education]
### Hi everyone,

Hello GitHub Support Team,

I hope this message finds you well. I am writing to seek assistance regarding my recent application for GitHub Copilot. Despite following all the necessary steps and providing the required documentation, my application was rejected. Below is a summary of the actions I have taken so far:

Provided Original School Certificate: I submitted my original school certificate of approval to learn for this year.
Two-Factor Authentication: I have enabled two-factor authentication on my account.
Updated Billing Information: My billing information is up-to-date and correctly entered.
Verified School Email: I verified my school email beforehand.
After submitting my school certificate, I received a notification that it was rejected because I have already used that certificate. I am confused by this response as I have not used the certificate for any other purpose.

Given the steps I have taken and the documentation I have provided, I kindly request your assistance in resolving this issue. I am eager to utilize GitHub Copilot for my academic pursuits and would greatly appreciate any help you can provide to rectify this situation.

Please let me know if any additional information or documentation is required from my end to facilitate the approval of my application.

Thank you for your time and support.

Best regards,

**Comment by SjxSubham** [comment]
try to upload ur school id card or library card as a valid proof....
(specially your id card)
it will don't show error..Thank me later :-)

**Comment by samus-aran** [comment]
Hi @OriyaU 👋 

If you have 2FA enabled and all your billing info is updated we can only suggest that you log out and log back in again and re-submit. Let us know how you get on. 👍
